GameServerLib/GameObjects/AttackableUnits/AttackableUnit.cs

```diff
@@ -30,15 +31,15 @@ public class AttackableUnit : GameObject, IAttackableUnit
         public IReplication Replication { get; protected set; }
 
         public AttackableUnit(
-            Game game,
+            IGame game,
             string model,
             IStats stats,
             int collisionRadius = 40,
             float x = 0,
             float y = 0,
             int visionRadius = 0,
             uint netId = 0
-        ) : base(game, x, y, collisionRadius, visionRadius, netId)
+        ) : base((Game)game, x, y, collisionRadius, visionRadius, netId)
```

Comment:
Goes with the one above.
```suggestion
        ) : base(game, x, y, collisionRadius, visionRadius, netId)
```

Reply:
Also fix here as suggested.